src/configs/curriculum.yml

```diff
@@ -15,6 +15,7 @@ files:
           "/**/part-[0-9][0-9][0-9].md",
           "/**/step-[0-9][0-9][0-9].md",
           "/**/15-javascript-algorithms-and-data-structures-22/**/*.md",
+          "/15-javascript-algorithms-and-data-structures-22/**/*.md",
```

Comment:
Does ignoring Superblocks maybe not work? You are trying to ignore one directory above the others.

Reply:
Right I'm trying this because the one above it did not work - the thought here is that `english` isn't being treated as a directory it's being treated as `/`, so there's nothing for `/**/` to match before the JS folder.